dev-tools/mage/gomod.go

```diff
@@ -33,64 +33,19 @@ type CopyModule struct {
 	FilesToCopy []string
 }
 
-var (
-	copyAll = []CopyModule{
-		CopyModule{
-			Name: "github.com/tsg/go-daemon",
-			FilesToCopy: []string{
-				"src",
-			},
-		},
-		CopyModule{
-			Name: "github.com/godror/godror",
-			FilesToCopy: []string{
-				"odpi",
-			},
-		},
-	}
-	filesToRemove = []string{
-		filepath.Join("github.com", "yuin", "gopher-lua", "parse", "Makefile"),
-		filepath.Join("github.com", "yuin", "gopher-lua", "parse", "parser.go.y"),
-	}
-)
-
 // Vendor cleans up go.mod and copies the files not carried over from modules cache.
-func Vendor() error {
+func TidyAndVerify() error {
```

Comment:
When is this target needed? Should we update any docs that were using `mage vendor` before?
And should it have a more specific name so it indicates that it is used to tidy and verify modules?

Reply:
It is just a leftover I was experimenting with. I was thinking about introducing a separate mage target for cleaning up go.mod. However, I abandoned the idea because it would be just `go mod tidy` and `go mod verify` with unnecessary overhead. Thanks for catching it.